src/wp-admin/includes/admin-filters.php

```diff
@@ -7,6 +7,9 @@
  * @since 4.3.0
  */
 
+
+global $pagenow;
+
```

Comment:
Why we need these global variable?

Reply:
Thanks for catching that. My initial approach was add to the `admin_init` hook when `$pagenow === 'press-this.php`, but that had other issues.

Removing.